snmp/datadog_checks/snmp/data/profiles/cisco-catalyst.yaml

```diff
@@ -0,0 +1,46 @@
+# Profile for Cisco Catalyst devices
+
+extends:
+  - _base.yaml
+  - _cisco-generic.yaml
+  
+sysobjectid:
+  - 1.3.6.1.4.1.9.1.241
+  - 1.3.6.1.4.1.9.1.1790
```

Comment:
The current sysobjectid of the Catalyst 3850 profile is not in these two, is it expected?

Besides according to [Wikipedia](https://en.wikipedia.org/wiki/Cisco_Catalyst) there are dozens of Catalyst devices, so I assume these two sysobjectids don't cover all of them?

(We may want to add a comment about the device each sysobjectid targets)

Reply:
It's expected not to have the Catalyst 3850 sysoid in `cisco-catalyst.yaml`, since this sysoid is already in the `cisco-3850.yaml` profile and we can't have the same sysoid twice.

There are indeed a few series, but they represent a lot of sysoid according to http://www.circitor.fr/Mibs/Mib/C/CISCO-PRODUCTS-MIB.mib. It was just to give some example, but maybe it should be more explicit.

Reply:
> It was just to give some example, but maybe it should be more explicit.

The `sysobjectid` is used to auto-assign profiles to devices, so IMO any device that is expected to be fit for this Catalyst profile should be listed here - not just a few for the sake of example. Of course customers can still create a new one that `extends` the Catalyst profile and provide their `sysobjectid` manually if it's not in the builtin list, but making device autodetection work in as many cases as possible is probably the best track.